doc/ref/debug.xml

```diff
@@ -223,8 +225,10 @@ If the first argument of an <Ref Func="Info"/> statement is a sum of
 Info classes, the handler of the first summand is used.
 <P/>
 The file or stream to which <Ref Func="Info"/> statements for individual
-<Ref Func="Info"/> classes print can be changed with 
-<Ref Func="SetInfoOutput"/>. The initial default for all <Ref Func="Info"/>
+<Ref Func="Info"/> classes print can be overriden with
+<Ref Func="SetInfoOutput"/>, retrieved with <Ref Func="InfoOutput"/>
+and reset to the default with <Ref Func="UnbindInfoOutput"/>.
```

Comment:
In the code I only see `UnbindInfoOutput`. A third alternative: `ResetInfoOutput`.

Just sayin', I don't really mind much either way.